README.md

```diff
@@ -0,0 +1,24 @@
+# edge-mc
+Edge Multi-Cluster
+
```

Comment:
I think we should start by saying that this is a subproject of kcp that aims to address the concerns of Edge Multi Cluster scenarios

Reply:
Also, I suggest we follow an outline similar to the main kcp project - something along the lines of 

- Overview
- Quickstart
- Next steps
- Contributing

We may not have yet code to try in Quickstart, but I think what might be of interest for developers browsing this site will be something to try out.